common/interface.h

```diff
@@ -0,0 +1,21 @@
+#ifndef __INTERFACE__
+#define __INTERFACE__
+
+#include <string>
+#include <net/if.h>
+
+namespace swss {
+
+inline bool isInterfaceNameLenOk(const std::string &ifaceName)
+{
+    if (ifaceName.length() >= IFNAMSIZ)
```

Comment:
@arfeigin the function should return True in case the validation was successful (ifName does not exceed the IFNAMSIZ). Is this a typo?

IMHO, should be the same as for CLI:
```python
def validate_interface_name_length(iface_name):
    """
    Verify that interface name length does not exceed IFNAMSIZ
    """
    if len(iface_name) == 0:
        return False
    return True if len(iface_name) < IFNAMSIZ else False
```

Can we reuse `swss-common` implementation also for CLI to avoid code duplication?

Reply:
Handled, thanks